scrapper.py

```diff
@@ -0,0 +1,120 @@
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+from tqdm import tqdm
+
+# Constants
+URL = "https://tradestat.commerce.gov.in/meidb/brc.asp?ie=i"
+DATABASE_FILE = "data.csv"
+
+# set the year range for data
+START_YEAR = 2020
+END_YEAR = 2022
+
+# set the month range for data
+START_MONTH = 1
+END_MONTH = 12
```

Comment:
Instead of defining these as constants, we should add scripts to take it only additional data and not process all years